plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/JdbcTableHandle.java

```diff
@@ -76,12 +78,25 @@ public JdbcTableHandle(SchemaTableName schemaTableName, RemoteTableName remoteTa
                 0);
     }
 
+    public JdbcTableHandle(
+            JdbcRelationHandle relationHandle,
+            TupleDomain<ColumnHandle> constraint,
+            Optional<List<JdbcSortItem>> sortOrder,
+            OptionalLong limit,
+            Optional<List<JdbcColumnHandle>> columns,
+            Set<SchemaTableName> otherReferencedTables,
+            int nextSyntheticColumnId)
+    {
+        this(relationHandle, constraint, sortOrder, limit, Optional.empty(), columns, otherReferencedTables, nextSyntheticColumnId);
+    }
+
     @JsonCreator
     public JdbcTableHandle(
             @JsonProperty("relationHandle") JdbcRelationHandle relationHandle,
             @JsonProperty("constraint") TupleDomain<ColumnHandle> constraint,
             @JsonProperty("sortOrder") Optional<List<JdbcSortItem>> sortOrder,
             @JsonProperty("limit") OptionalLong limit,
+            @JsonProperty("comment") Optional<String> comment,
```

Comment:
This would better fit `JdbcNamedRelationHandle`.
But even better if not part of table handle at all. 
See https://github.com/trinodb/trino/pull/11104 for example of a problem that was caused by additional metadata in a handle class.